work/covidvu/predict.py

```diff
@@ -392,8 +392,8 @@ def predictRegions(regionTrainIndex,
             _dumpRegionPrediction(prediction, siteData, predictionsPercentiles)
         elif subGroup == 'casesUSStates':
             _dumpRegionPrediction(prediction, siteData, predictionsPercentiles,
-                                  meanFilename='prediction-mean-USStates-%s.json',
-                                  confIntFilename='prediction-conf-int-USStates-%s.json',)
+                                  meanFilename='prediction-US-mean-%s.json',
```

Comment:
@jaryaman - Humble suggestion:  define these string templates as constants at the top of the source file if they appear in more than one place.  It makes it easier to identify them.  The only place where we may not always do that is in `test_whatever.py` unit tests, because magic strings with specific values (e.g. `"bogus-file-name.json"`) may be part of the test.